network/src/utils.rs

```diff
@@ -0,0 +1,60 @@
+use substreams::scalar::BigInt;
+use substreams_ethereum::pb::eth::v2::{self as eth};
+use substreams_ethereum::scalar::BigIntSign;
+use crate::pb::aggregate_data;
+
+/// Returns the timestamp for the start of the most recent day
+pub(crate) fn get_latest_day(timestamp: i64) -> i64 {
```

Comment:
can you move the functions here that may be used across different substreams into `substreams-helper`?

Reply:
Sure 👍